courses/management/commands/transfer_enrollment.py

```diff
@@ -0,0 +1,82 @@
+"""Management command to change enrollment status"""
+from django.contrib.auth import get_user_model
+
+from courses.management.utils import EnrollmentChangeCommand, fetch_user
+from courses.constants import ENROLL_CHANGE_STATUS_TRANSFERRED
+from courses.models import CourseRunEnrollment, ProgramEnrollment
+
+User = get_user_model()
+
+
+class Command(EnrollmentChangeCommand):
+    """Sets a user's enrollment to 'transferred' and creates an enrollment for a different user"""
+
+    help = "Sets a user's enrollment to 'transferred' and creates an enrollment for a different user"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--from-user",
+            type=str,
+            help="The id, email, or username of the enrolled User",
+        )
+        parser.add_argument(
+            "--to-user",
+            type=str,
+            help="The id, email, or username of the User to whom the enrollment will be transferred",
+        )
+        group = parser.add_mutually_exclusive_group()
+        group.add_argument(
+            "--program",
+            type=str,
+            help="The 'readable_id' value for an enrolled Program",
+        )
+        group.add_argument(
+            "--run",
+            type=str,
+            help="The 'courseware_id' value for an enrolled CourseRun",
+        )
+        super().add_arguments(parser)
+
+    def handle(self, *args, **options):
+        from_user = fetch_user(options["from_user"])
+        to_user = fetch_user(options["to_user"])
+        enrollment, enrolled_obj = self.fetch_enrollment(from_user, options)
+
+        if options["program"]:
+            ProgramEnrollment.objects.create(
```

Comment:
I think we need to create `CourseRunEnrollment` for the associated runs here for the new user

Reply:
👍 knew I was forgetting something. All set now